js/src/lib/components/InferenceWidget/widgets/ImageToImageWidget/ImageToImageWidget.svelte

```diff
@@ -94,9 +94,9 @@
 		}
 
 		const requestBody = {
-			image: imageBase64,
+			inputs: imageBase64,
```

Comment:
hi @mishig25, considering last review from @Narsil we might not need to rename this inputs to image, actually both should work. However the parameters, prompts are correctly changed!

Reply:
Either way works for me.
`inputs` is more standardized across the board so I'm in favor of the change.

The API however must accept both (for backward compatibility)